includes/wf_crm_webform_preprocess.inc

```diff
@@ -179,7 +179,12 @@ class wf_crm_webform_preprocess extends wf_crm_webform_base {
         'scope' => 'footer',
       );
       $page_id = $this->data['contribution'][1]['contribution'][1]['contribution_page_id'];
-      $js_vars['contributionCallback'] = url('civicrm/contribute/transact', array('query' => array('reset' => 1, 'id' => $page_id, 'qfKey' => $this->getQfKey(), 'snippet' => 4), 'alias' => TRUE));
+      $contributionCallbackQuery = array('reset' => 1, 'id' => $page_id, 'qfKey' => $this->getQfKey(), 'snippet' => 4);
+      if ($this->data['contribution'][1]['contribution'][1]['is_test'] == '1') {
```

Comment:
let's play it safe and change this line to 

    if (!empty($this->data['contribution'][1]['contribution'][1]['is_test'])) {